numpy/lib/arraysetops.py

```diff
@@ -311,11 +311,21 @@ def intersect1d(ar1, ar2, assume_unique=False):
     assume_unique : bool
         If True, the input arrays are both assumed to be unique, which
         can speed up the calculation.  Default is False.
-
+    return_indices : bool
+        If True, the indices which correspond to the intersection of the 
+        two arrays are returned. The first instance of a value is used 
+        if there are multiple. Default is False. 
+    
+        .. versionadded:: 1.15.0    
+        
     Returns
     -------
     intersect1d : ndarray
         Sorted 1D array of common and unique elements.
+    comm1 : ndarray
+        Indices of common elements in ar1. 
```

Comment:
You should try to match the docs for `np.unique` here, and document this as

    The indices of the first occurrences of the common values in `arr1`. Only provided if `return_indices` is True.